src/plugins/data_table/data_table.js

```diff
@@ -2,30 +2,167 @@ var LocalDataTable = (function() {
 
   var Base = Backdraft.plugin("Base");
 
+  var DEFAULT_JST_DELIMS = {
+    evaluate    : /<%([\s\S]+?)%>/g,
+    interpolate : /<%=([\s\S]+?)%>/g,
+    escape      : /<%-([\s\S]+?)%>/g
+  };
+
+  var DataTableFilter = Base.View.extend({
+    filterTemplate : _.template('\
+      <div class="toggle-filter-button" data-toggle="dropdown">\
+        <span class="<%= filterButtonClass %>"></span>\
+      </div>\
+    ', null, DEFAULT_JST_DELIMS),
+
+    filterMenuTemplate : _.template('\
```

Comment:
I kind of liked having the filters all in a single template and having the menu separate.  What is the reasoning for breaking out individual filters instead of the entire filter menu?